src/main/frontend/app/functions/validateForm.js

```diff
@@ -0,0 +1,43 @@
+const validateForm = (requiredFields, stateGetter, errorSetter) => {
+  let submitErrors = {};
+  requiredFields.forEach((field) => {
+    if (stateGetter[field].trim() === "") {
+      submitErrors = {
+        ...submitErrors,
+        [field]: "is required.",
+      };
+    }
+  });
+
+//   const validPhoneNo = /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/;
+//   if (
+//     !validPhoneNo.test(stateGetter["phoneNumber"]) &&
+//     stateGetter["phoneNumber"].trim() !== ""
+//   ) {
+//     submitErrors = {
+//       ...submitErrors,
+//       ["phoneNumber"]:
+//         "must be in valid format: xxx-xxx-xxxx, xxx.xxx.xxxx, xxx xxx xxxx",
+//     };
+//     console.log("is correct");
+//   }
+
+//   const validEmail = /^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$/;
+//   if (
+//     !validEmail.test(stateGetter["email"]) &&
+//     stateGetter["email"].trim() !== ""
+//   ) {
+//     submitErrors = {
+//       ...submitErrors,
+//       ["email"]: "must be in valid format.",
+//     };
+//   }

```

Comment:
You should not have commented out code in your PR commits.

Reply:
Updated.